.golangci.yml

```diff
@@ -2,47 +2,105 @@ run:
   deadline: 5m
 
 linters:
+  fast: false
   disable-all: true
   enable:
-    #- bodyclose
+   # Enabled by Default
     - deadcode
+    - errcheck
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - typecheck
+    - unused
+    - varcheck
+   # Disabled by Default
+    #- asasalint
+    #- asciicheck
```

Comment:
I didn't realize our `.golangci.yml` had abunch of rules commented out. That seems weird. And makes the linter file harder to read: I don't think anyone really needs to know what rules we explicitly aren't using, that's implied by them not being in the file to begin with.

Can we just remove them?